docs/configuration_file.md

```diff
@@ -155,6 +155,25 @@ thread_count = <em>&lt;thread_count&gt;</em>
 
 Note: the `thread_count` parameter is optional and denotes the number of parallel threads used to by the plugin to connect to the endpoint. The higher this number, the faster the execution, however if the number is too high the server may interrupt the connection with a rate-limit error.
 
+### Kraken Section (REST)
+This plugin is REST-based and requires setting up API Keys in your Kraken account settings (click on the API link).
+
+**IMPORTANT NOTE**:
+* when setting up API key/secret/passphrase, only use read permissions (DaLI does NOT need write permissions);
+* store your API key, secret and passphrase safely and NEVER share it with anyone!
```

Comment:
Does Kraken have passphrase?

Reply:
I don't think it does. I didn't have to make one.

Reply:
So we should remove reference to passphrase from the docs.

Reply:
Whoops totally missed that.